### DISCUSSION TITLE
Running WiX heat inside a linux docker container

### DISCUSSION THREAD

**Discussion by Shevchenko-Alexander** [wixtoolset/issues#7610] [Questions] [answered]
Hello!
We are trying to move our installer generation to WiX and it works perfect so far.
But now we are facing the following issue. For the CI/CD we are using a lunux based docker container and get the following error message:

```
heat.exe : error HEAT0001: System.DllNotFoundException: Unable to load shared library 'kernel32.dll' or one of its dependencies. In order to help diagnose loading problems, consider using a tool like strace. If you're using glibc, consider setting the LD_DEBUG environment variable: [/streamer/src/hub/InstallerGenerator/InstallerGenerator.wixproj]

Build FAILED.
```

Do you know how can we workaround that? Does it mean, that by using heat we can't use linux based container anymore?

**Comment by robmen** [comment, marked as answer]
WiX doesn't support Linux today. We had some interest in WiX v4 but no one picked up the necessary work. What are you doing to make any of it work today?

**Reply by Shevchenko-Alexander** [reply]
Thank you for the fast clear answer Rob! And thank you for your weekly YouTube series too
